src/templates/community-champion-profile.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import Breadcrumb from '~components/dev-hub/breadcrumb';
+import Layout from '~components/dev-hub/layout';
+import { fontSize, screenSize, size } from '~components/dev-hub/theme';
+
+const BREADCRUMB_BOTTOM_MARGIN = '40px';
+
+const ContentContainer = styled('div')`
+    padding: ${size.default} ${size.xxlarge};
+    @media ${screenSize.upToLarge} {
+        padding: ${size.mediumLarge} ${size.default};
+    }
+`;
+
+const StyledBreadcrumb = styled(Breadcrumb)`
+    line-height: 1;
```

Comment:
This is clever, I've never seen it before. Nice!

Reply:
was not my idea, i took it from here 😛  https://github.com/mongodb/devhub/blob/staging/src/components/dev-hub/hero-banner.js#L67

Reply:
Ah yes, Gregg had some tricks like that up his sleeve